src/commands/codepush/lib/legacy-codepush-service-client.ts

```diff
@@ -0,0 +1,108 @@
+import * as Request from "request";
```

Comment:
nitpick: This should be imported as `request`, not `Request`. Upper case is for class constructors, request is a regular function.